notebooks/source/bayesian_hierarchical_stacking.ipynb

```diff
@@ -21,7 +21,7 @@
     "  <figcaption>Photo by Belinda Fewings, https://unsplash.com/photos/6p-KtXCBGNw.</figcaption>\n",
     "</figure> \n",
     "\n",
-    "### Table of Contents\n",
+    "## Table of Contents\n",
```

Comment:
This one makes the tutorial sections not [arranged correctly](https://num.pyro.ai/en/latest/tutorials/bayesian_hierarchical_stacking.html#Table-of-Contents).